configs/sst_virtualization-virt.yaml

```diff
@@ -0,0 +1,97 @@
+document: feedback-pipeline-workload
+version: 1
+data:
+  name: Virtualization on RHEL - KVM Userspace
+  description: Userspace agents and tools that enable virtualization capabilities on RHEL
+  maintainer: sst_virtualization
+
+  modules_enable:
+  - virt:rhel
+
+  packages:
+  - amtterm
+  - augeas
```

Comment:
augeas is part of the SST because I'm its maintainer. augeas is in BaseOS, outside of any module.

Reply:
The BaseOS / AppStream split will be happening later. 
So this will stay here for now.